README.md

```diff
@@ -74,6 +74,47 @@ Finally, the client can produce the output[s] of the OPRF protocol using the ser
 const [output] = await client.finalize(finData, evaluation);
 ```
 
+### Support for @noble Crypto backend (faster & ristretto/decaf)
+
+With this library, you have the flexibility to switch out the cryptographic
+backend to`@cloudflare/voprf-ts/crypto-noble`. It has much better performance
+and provides support for Ristretto and Decaf groups:
+
+```js
+Oprf.Suite.RISTRETTO255_SHA512
+Oprf.Suite.DECAF448_SHAKE256
+```
+
+Before doing so, be aware of the following:
+
+- The `@noble/curves` library uses native JavaScript `BigInt` for arithmetic
+  operations, which are non-constant time by nature. More importantly, the
+  noble libraries use constant time(CT) algorithms.
```

Comment:
This is the important thing to highlight. 

```suggestion
- The `@noble/curves` library uses native JavaScript `BigInt` for arithmetic
  operations, which are non-constant time by nature.
```

Reply:
Your current library is also not constant-time. Why don’t you highlight this fact as well?

Reply:
@armfazh 

> More importantly, the noble libraries use constant time(CT) algorithms.

I see you removed this.

Perhaps we can instead go with: "However, noble libraries DO use constant time(CT) algorithms." 

That is simply stating a fact, no?

Reply:
- Arithmetic operations in the `@noble/curves` library leverage native JavaScript's `BigInt`. 
  - These operations inherently operate in non-constant time. 
  - However, noble libraries DO take care to employ constant time (CT) algorithms.
  - In the interest of fairness, it must be noted that `sjcl` is also not perfectly CT either. 
  - As always, **users are encouraged to understand the implications of their chosen cryptographic backend** fully. 

Probably no one is going to be entirely happy with that, but it seems fair-er

Reply:
![image](https://github.com/cloudflare/voprf-ts/assets/525211/a5382a56-2ff9-42c6-a1b6-368e5587eb0e)

We clearly all want this and we're !so close! - deep breaths

Reply:
Added:   - This is a community contribution and Cloudflare is still researching a perfectly CT JavaScript BN implementation

Reply:
> About the documentation, I think it deserves a full document Security.md that summarizes > the current status of both back-ends.
> I would like to write this, and share with you'l when ready.

ack